src/helpers/index.ts

```diff
@@ -15,3 +16,11 @@ export const replaceArray = (string: string, find: ReplaceArrayDictionary): stri
  * @type regex
  */
 export const ArabicContextualForms = /[ي|ﻱ|ﻲ|ﻚ|ك|ﻚ|ﺔ|ﺓ|ة]/g;
+
+export function isString(value: string): boolean {
+	return {}.toString.call(value) === "[object String]";
```

Comment:
Please create a function:
```js
function getType(value: any): string {
   return Object.prototype.toString.call(value).slice(8, -1).toLowerCase();
}
```
And then use this function in `isString` or `isNumber` functions as below:
```js
function isNumber(value: any): boolean {
    return getType(value) === "number";
}
```